src/app/_theme-dark.scss

```diff
@@ -0,0 +1,96 @@
+// Dark theme
+.theme-dark {
+  // Base colours
+  --theme-accent-primary: #ff3f00; // Interaction styles (hover, active, focus)
+  --theme-accent-secondary: #68a0b7; // Secondary interactions
+
+  // More vibrant P3 colourspace tints for displays that support them
+  @supports (color: color(display-p3 1 1 1)) {
+    --theme-accent-primary: oklch(0.66 0.24 37.65);
+  }
+
+  // Generic fills for modal backgrounds (see below)
+  --theme-fill-modal: #000; // Base fill for popups, drawers, dropdowns etc
+  --theme-fill-modal-actions: #161616; // Layered/contextual modals (search, item actions)
+
+  // Text
+  --theme-text: #ccc;
+  --theme-text-invert: #fff;
+  --theme-text-secondary: #777;
+
+  // Buttons
+  --theme-button-bg: rgba(255, 255, 255, 0.2);
+  --theme-button-text: white;
+
+  // Input field (text boxes and text areas)
+  --theme-input-bg: #333;
+
+  // Shadows
+  --theme-drop-shadow: 0 0 18px 0 #000;
+
+  // Tooltips
+  --theme-tooltip-underline: var(--theme-accent-primary);
+  --theme-tooltip-header-bg: #000;
+  --theme-tooltip-body-bg: #1a1a1a;
+  --theme-tooltip-border: #555;
+  --theme-tooltip-minimal-bg: #5d5970;
+
+  // App background gradient pattern
+  --theme-app-bg-gradient-start: #1a1a1a;
+  --theme-app-bg-gradient-end: #111;
+  --theme-app-bg-gradient: radial-gradient(
+    circle at 50% 70px,
+    var(--theme-app-bg-gradient-start) 0%,
+    var(--theme-app-bg-gradient-end) 100%
+  );
+
+  // App
+  --theme-app-bg: var(--theme-app-bg-gradient);
+  // The color of PWA elements like the title bar. This cannot be a gradient.
+  --theme-pwa-background: #000;
+
+  // Header
+  --theme-header-nav-bg: black;
+  --theme-header-nav-slideout-menu-bg: var(--theme-fill-modal);
+  --theme-header-characters-bg: rgba(15, 15, 15, 0.9);
+  --theme-header-characters-txt: #aaa;
+
+  // Dropdown menu
+  --theme-dropdown-menu-bg: var(--theme-fill-modal);
+
+  // Search
+  --theme-search-bg: rgba(255, 255, 255, 0.15);
+  --theme-search-dropdown-bg: var(--theme-fill-modal-actions);
+  --theme-search-dropdown-border: rgba(255, 255, 255, 0.2);
+
+  // Items 'polaroid' framed icons
+  --theme-item-polaroid: #456;
+  --theme-item-polaroid-txt: white;
+  --theme-item-polaroid-masterwork: black;
```

Comment:
I still think this feels weird - the masterworks stand out, but they lose all separation with the background.